src/w3c/templates/headers.js

```diff
@@ -135,7 +109,7 @@ function getSpecSubTitleElem(conf) {
 export default conf => {
   return html`
     <div class="head">
-      ${conf.logos.map(showLogo)} ${getSpecTitleElem(conf)}
+      ${conf.logos.map(showLogo)} ${updateSpecTitleElem(conf)}
```

Comment:
I don't like how several modules are "updating" the title. Maybe we should create `core/title` to update it first?

Reply:
Yeah, that might be the way to go. Let's get the logic right first, get everything passing, then refactor.